assets/queries/terraform/azure/aks_uses_azure_policies_addon/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "43789711-161b-4708-b5bb-9d1c626f7492",
+  "queryName": "AKS Uses Azure Policies Add-On",
```

Comment:
The query name could be more intuitive like "AKS Azure Policies Add-On Disabled" (just a suggestion)

Reply:
I agree with the naming suggestion, please make sure to change the directory name to match the query name